src/plugins/deps/geomap-lib.js

```diff
@@ -1162,7 +1165,8 @@ var componentName = "wb-geomap",
 		var aoiDiv = $( "#geomap-aoi-" + geomap.id ),
 			extent, left, bottom, right, top, geomProj;
 
-		aoiDiv.append( "<fieldset id='form-aoi-" + geomap.id + "'>" +
+		if ( mapContainerWidth > 600 ) {
```

Comment:
Just put the numbers into variables instead of duplicating the append statement.